modules/migration-configuring-mcg.adoc

```diff
@@ -11,30 +11,38 @@
 = Retrieving Multicloud Object Gateway credentials
 
 ifdef::installing-3-4,installing-mtc[]
-You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint in order to configure MCG as a replication repository for the {mtc-full} ({mtc-short}).
+You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint to configure MCG as a replication repository for the {mtc-full} ({mtc-short})
+
+You must retrieve the Multicloud Object Gateway (MCG) credentials to create a `Secret` custom resource (CR) for {mtc-short}.
+endif::[]
+
+ifdef::installing-oadp-mcg[]
+You must retrieve the Multicloud Object Gateway (MCG) credentials to create a `Secret` custom resource (CR) for the OpenShift API for Data Protection (OADP).
```

Comment:
```suggestion
You must retrieve the Multicloud Object Gateway (MCG) credentials in order to create a `Secret` custom resource (CR) for {mtc-short}.
```

Reply:
@mburke5678 This is the text for OADP. There's a separate text for MTC.

Reply:
@RichardHoch Sorry! I cut and pasted my [suggestion from the previous](https://github.com/openshift/openshift-docs/pull/78361#pullrequestreview-2180828308).  This is what I intended:

```suggestion
You must retrieve the Multicloud Object Gateway (MCG) credentials in order to create a `Secret` custom resource (CR) for the OpenShift API for Data Protection (OADP).
```